### PR TITLE
Add user influenced prompt on transcript generation

### LINKED ISSUE #190: feat: user-influenced prompt definition
In settings/onboarding, have cards to let the user choose among some predefined, alternative LLM prompts. 

In the backend, already implemented is a field for user-defined prompts, and the frontend can modify it while the backend is adapted to respond to it.

### PR DESCRIPTION
**Pull request by pmdlt**
### Description
This pull request introduces user-influenced prompt definitions to personalize the news feed experience. Users can now select predefined prompts to tailor how the news is presented to them.

### Changes Made
- Added predefined user prompts to localization files (`app_en.arb`, `app_fr.arb`).
- Updated `NewsSettings` model to include `userPrompt` attribute.
- Modified `NewsSettingsViewModel` to handle `userPrompt`.
- Updated `AccessibilityView` and `InterestWizardView` to include UI elements for selecting predefined prompts.
- Added dependency on the `choice` package for prompt selection UI component.
- Updated Supabase functions to pass `userPrompt` to transcript generation.

### New Features
- Users can personalize their news feed by selecting from predefined prompts:
  - As if you were 5 years old
  - As if you were a radio host
  - As if you were a robot
  - With a lot of enthusiasm
  - With a lot of sarcasm
  - With a bit of humor

### Related Issue
- Closes #190

### Checklist
- [x] Added user prompt options to localization files.
- [x] Updated models and view models to include user prompts.
- [x] Updated views to allow prompt selection.
- [x] Ensured new functionality is reflected in Supabase functions.
- [x] Added necessary dependencies.

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -74,6 +74,13 @@
       }
     }
   },
+  "explainingUserPrompt": "You can personalize your news feed with this predefined prompts.\n - Read the news...",
+  "prompt1": "as if you were 5 years old",
+  "prompt2": "as if you were a radio host",
```
[the remark this answers is not in the document]
Comment (pmdlt) [reply lib/l10n/app_en.arb]: Yes, thanks for pointing that out! It's edited :)

File: lib/l10n/app_fr.arb [hunk 2]
```diff
@@ -54,6 +54,13 @@
     "alarmLoop": "Répétez l'audio",
     "alarmNotSet": "Alarme désactivée",
     "alarmSetFor": "Alarme activée pour {date} prochain",
+    "explainingUserPrompt": "Vous pouvez personnaliser votre fil d'actualité grâce à ces prompts prédéfinis.\n - Lire les actualités...",
+    "prompt1": "comme si tu avais 5ans",
+    "prompt2": "comme si tu étais un robot",
```
[the remark this answers is not in the document]
Comment (pmdlt) [reply lib/l10n/app_fr.arb]: Yes, thanks for pointing that out! It's edited :)

**Comment by pmdlt** [comment]
> Great work ! I think it would be good to have a "clear" button to deselect the prompt. Also, I've got an issue where the `Cancel` button cannot be tapped, both on Web and Android.

Thanks for the review!
As said in private, we'll do what you mentioned, I reversed the prompts, so “no need” for that button, the “default” prompt when clicked will be the radio host.

**Review by Thechi2000 — APPROVED** [review]
Let's merge that !

**Review by quartztz — APPROVED** [review]
Amazing feature ! Better merge it since we promised it this morning :D

**Pull request merged**